core/scripts/sql/update/14.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS cluster (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    owner_id INT UNSIGNED NOT NULL,
```

Comment:
Since we are creating a general purpose cluster, its better to memorize identifiers(URL) of the machine, also some metadata like provider(e.g. `aws`).

Reply:
Since the cloudaligner only supports `aws`, I think it is better to not have a `provider` column for now. If we also support other providers, definitely need this column.

Reply:
For the identifiers, I will add it in the next PR.